libs/ui-toolkit/arrow-variation/src/lib/arrow-variation.component.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('ArrowVariationComponent', () => {
 
   describe('positive values', () => {
     it('arrow direction must be `up`', () => {
-      component.updateArrow(2019);
+      component.updateArrow();
```

Comment:
Since you have changed `updateArrow`, you'll now need to pass these values to the component using the `value` input. It won't be necessary to call the `updateArrow` method directly, since its called in the `set` function of the input.